llvm/utils/vim/ftplugin/llvm.vim

```diff
@@ -11,3 +11,4 @@ setlocal softtabstop=2 shiftwidth=2
 setlocal expandtab
 setlocal comments+=:;
 setlocal commentstring=;\ %s
+setlocal iskeyword=%,@-@,$,a-z,A-Z,48-57,_,.,-
```

Comment:
Is `48-57` really how one encodes `0-9`? If so, ew!

Reply:
Haha yep! I could add a comment to explain the list? It's a little cryptic.

Reply:
I've added a comment